romancal/regtest/test_skycell_generation.py

```diff
@@ -13,12 +13,21 @@
 pytestmark = [pytest.mark.bigdata]
 
 EXPECTED_FILENAMES = [
-    "r00001_p_v01001001001_r274dp63x31y80_f158_asn.json",
-    "r00001_p_v01001001001_r274dp63x31y81_f158_asn.json",
-    "r00001_p_v01001001001_r274dp63x32y82_f158_asn.json",
-    "r00001_p_v01001001001_r274dp63x32y80_f158_asn.json",
-    "r00001_p_v01001001001_r274dp63x32y81_f158_asn.json",
-    "r00001_p_v01001001001_r274dp63x32y82_f158_asn.json",
+    "r00001_p_v01001001001_270p65x48y69_f158_asn.json",
+    "r00001_p_v01001001001_270p65x48y70_f158_asn.json",
+    "r00001_p_v01001001001_270p65x48y71_f158_asn.json",
+    "r00001_p_v01001001001_270p65x49y69_f158_asn.json",
+    "r00001_p_v01001001001_270p65x49y70_f158_asn.json",
+    "r00001_p_v01001001001_270p65x49y71_f158_asn.json",
+    "r00001_p_v01001001001_270p65x50y69_f158_asn.json",
+    "r00001_p_v01001001001_270p65x50y70_f158_asn.json",
+    "r00001_p_v01001001001_270p65x50y71_f158_asn.json",
+    "r00001_p_v01001001001_270p65x51y69_f158_asn.json",
+    "r00001_p_v01001001001_270p65x51y70_f158_asn.json",
+    "r00001_p_v01001001001_270p65x51y71_f158_asn.json",
+    "r00001_p_v01001001001_270p65x52y69_f158_asn.json",
+    "r00001_p_v01001001001_270p65x52y70_f158_asn.json",
+    "r00001_p_v01001001001_270p65x52y71_f158_asn.json",
```

Comment:
This is more like what I expect in terms of more new files than old files due to the smaller patch sizes.